securedrop_client/gui/widgets.py

```diff
@@ -1018,21 +1018,32 @@ def setup(self, controller):
         """
         self.controller = controller
         self.star.setup(self.controller)
+        self.controller.message_ready.connect(self.set_snippet)
+        self.controller.reply_ready.connect(self.set_snippet)
+        self.controller.reply_succeeded.connect(self.set_snippet)
+        self.controller.file_ready.connect(self.set_snippet)
 
```

Comment:
headsup: you may need to connect one more signal if [this](https://github.com/freedomofpress/securedrop-client/pull/724/files#diff-16a175b4662541f3889b432ac9e7e0e8R1942) gets merged first